packages/chrome.extensions.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,5 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+$regKeyPath = "HKLM:\SOFTWARE\WOW6432Node\Policies\Google\Chrome\ExtensionInstallForcelist"
+Remove-Item -Path $regKeyPath -Recurse -Force -ea 0
```

Comment:
This could also remove modifications made by the user and does not remove the installed extension. I think It is ok to not have a `chocolateyuninstall.ps1` file in this case.

Reply:
Could the `ExtensionInstallForcelist` include other extensions apart from the ones we install here? I think it could and to avoid risking deleting other installed extension, I would delete this file (remember [uninstallation is best effort](https://github.com/mandiant/VM-Packages/wiki/Coding-Conventions#uninstalling--updating-are-best-effort)).

Reply:
No, it's not possible to add extensions using `ExtensionInstallForcelist` via Chrome (it should be added manually via Registry).
User-installed extensions goes under a different path, I just tested it and the user-installed extensions are not effected by the modifications done to `ExtensionInstallForcelist` (installing/uninstalling this package).
It's worthy to mention that by default the `ExtensionInstallForcelist` key doesn't exist.